packages/dds/merge-tree/src/mergeTree.ts

```diff
@@ -593,6 +593,8 @@ export class MergeTree {
 	public mergeTreeDeltaCallback?: MergeTreeDeltaCallback;
 	public mergeTreeMaintenanceCallback?: MergeTreeMaintenanceCallback;
 
+	// TODO:AB#29553: This property doesn't seem to be adequately round-tripped through summarization.
+	// Specifically, it seems like we drop information about obliterates within the collab window for at least V1 summaries.
 	private readonly obliterates = new Obliterates(this);
```

Comment:
hmmm. yeah. you are right about V1, but today only matrix uses v1, and doesn't use obliterate. for the legacy format this should be fine, as all ops in the collab window get replayed.

Reply:
Yeah. I think it ends up being mostly fine for legacy snapshot. Still want to audit a few things though as there are some tricky things in this area..